tikv-client/src/main/java/com/pingcap/tikv/policy/RetryPolicy.java

```diff
@@ -62,11 +64,9 @@ public RespT callWithRetry(Callable<RespT> proc, String methodName) throws GrpcE
       }
 
       // Handle response error
-      if (handler != null) {
-        boolean retry = handler.handleResponseError(backOffer, result);
-        if (retry) {
-          continue;
-        }
+      boolean retry = handler.handleResponseError(backOffer, result);
```

Comment:
```
if(handler.handleResponseError(backOffer, result)) {
    continue; 
}
```  
?

Reply:
code is more understandable with `boolean retry`